### ISSUE TITLE
Guidelines for Human-AI Interaction

### ISSUE THREAD

**Issue by idigitopia** [idigitopia/idigitopia.github.io#3]
[paper](http://erichorvitz.com/Guidelines_Human_AI_Interaction.pdf) [demo](https://aidemos.microsoft.com/guidelines-for-human-ai-interaction/demo)

- Context: 
    - AI is probabilistic and prone to errors. (For example weather reports are not always accurate, and probabilistic in nature. 40% chance of rainfall) 
    - AI learns and changes over time. (For example, My youtube recommendations are never of the same quality)
    - AI can violate established principles of traditional UI Design. (For example conditions, consistency)
- Notion: "Issues caused by AI frequently can't and don't need to be solved by more fancy algorithms, but by good design that mitigates them."
- Motivation: We wish to go from Principia to pragmatism, converting an extensive body of literature for designing for human-AI interaction to 18 actionable guidelines.  
- Guidelines: 
    - Initially: When the UI is being designed. 
    - During Interaction: When the user interacts with the UI. 
    - When wrong: When the output of the AI is wrong. 
    - Over time: Human - AI interaction over a period of time. 
- "When wrong" is probably the most important section for explainability and interpretability as it is more crucial to be able to interact with the AI better when it is wrong or harder to understand. 
- In summary, these guidelines help one focus on the Human needs of AI design.
